src/ert/ensemble_evaluator/evaluator.py

```diff
@@ -417,6 +421,9 @@ async def _start_running(self) -> None:
         ]
         # now we wait for the server to actually start
         await self._server_started.wait()
+        # setup message queues
+        self._ensemble.scheduler_queue = self._events
+        self._ensemble.manifest_queue = self._manifest_queue
```

Comment:
Should this be moved to the constructor?

Reply:
No, it can't as these queues are passed by the evaluator.